docs/source/requests.md

```diff
@@ -1723,30 +1724,42 @@ A client will receive NACK for
 Setting (enabling/disabling) a transaction author agreement for the pool.
 If transaction author agreement is set, then all write requests to Domain ledger (transactions) must include additional metadata pointing to the latest transaction author agreement's digest which is signed by the transaction author.
 
-If no transaction author agreement is set, or it's disabled, then no additional metadata is required.
+For any given version of transaction author agreement text and ratification date cannot be changed once set. Ratification date cannot be in future.
```

Comment:
I would exchange these two paragraphs since the second one is related to the very first one (client's metadata in request).